Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-core/kodi-main/) for Kodi core/kodi main.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-core/-/kodi-main/horizontal-auto.svg)

gade01 commented:
This PR should be left open.
This is to test what Weblate changes.
Weblate will push to this PR once every week until merged or closed.

weblate committed:
Added translation using Weblate (Sicilian)

Added translation using Weblate (Kannada (India))

Added translation using Weblate (Moldavian (Romania))

Added translation using Weblate (Asturian (Spain))

Update translation files

Updated by "Source update" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main/
Translation: Kodi core/kodi main
